File: csharp/ex061.cs
// Crie um programa que mostre na tela a seguinte contagem, usando a estrutura
// “faça enquanto”
// 0 3 6 9 12 15 18 21 24 27 30 Acabou!


class Ex061
{
  public int counter = 0;

  public void FirstCountdown()
  {
    Console.Write(counter);
    do
    {
      counter += 3;
      Console.Write($" {counter} ");

    } while (counter < 30);
    Console.Write(" Acabou!");
  }
}

File: python/ex061.py
# Crie um programa que mostre na tela a seguinte contagem, usando a estrutura
# “faça enquanto”
# 0 3 6 9 12 15 18 21 24 27 30 Acabou!

counter = 0 
while True:
    print(counter, end=" ")
    counter += 3
    if(counter > 30):
        break
print("Acabou!")


